src/pages/User/index.js

```diff
@@ -9,10 +9,21 @@ import PullRequests from './components/PullRequests';
  * User Component.
  */
 export class User extends Component {
-  state = {
```

Comment:
Let's keep this. It's fine to initialize the `state` like this.

Reply:
ok dai.